tasks/main.yml

```diff
@@ -7,6 +10,10 @@
        dest={{ otp_bin_dir }}
        version={{ otp_version }}
 
+- name: Set OpenTripPlanner Session Timeout
+  lineinfile: dest={{ otp_bin_dir }}/src/client/WEB-INF/web.xml regexp="(^|\W)<session-timeout>30</session-timeout>$" line="\t\t<session-timeout>{{ otp_session_timeout }}</session-timeout>"
```

Comment:
`web.xml` is otherwise a totally uninteresting file; not sure why this one parameter got buried in it.  I think it's very unlikely we'd want to mess with any of the rest of it.